lib/reporters/html.js

```diff
@@ -219,7 +235,6 @@ function HTML(runner, options) {
   }
 
   function updateStats() {
-    // TODO: add to stats
```

Comment:
[Praise] Nice to see a TODO resolved 😄 I hadn't even noticed this one!

Reply:
note I just removed this, it looked completed and was from 2011 and it was bothering me :D